src/test/java/domain/frame/NormalFrameTest.java

```diff
@@ -0,0 +1,42 @@
+package domain.frame;
+
+import domain.Pins;
+import domain.state.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class NormalFrameTest {
+
+    private NormalFrame frame;
+
+    @BeforeEach
+    void setUp() {
+        frame = new NormalFrame();
+    }
+
+    @Test
+    void strike() {
+        State state = frame.setKnockedDownPins(Pins.ALL);
+        assertThat(state instanceof Strike).isTrue();
```

Comment:
`Assert#isInstanceOf(Class)`을 사용해 보면 어떨까요?

```suggestion
        assertThat(state).isInstanceOf(Strike.class);
```

Reply:
👍